src/NetPanzer/Core/main.cpp

```diff
@@ -92,7 +92,10 @@ UnitProfileInterface::clearProfiles();
 
     SDL_Quit();
     if(save_config && gameconfig) {
+        LOGGER.info("Saving game config.");
         gameconfig->saveConfig();
+    } else if (!gameconfig) {
+        LOGGER.warning("No game config to save.");
```

Comment:
this is just a useful log incase something bad happens, but should normally never happen